\documentclass[11pt]{article}
\usepackage{array, xcolor, lipsum, bibentry, parallel}
\usepackage[margin=3cm]{geometry}
\usepackage{setspace}
\usepackage{tabularx}
\usepackage{paralist}
\usepackage{scrextend}
\usepackage[colorlinks = true, linkcolor = blue, urlcolor  = blue, citecolor = blue, anchorcolor = blue]{hyperref}


\title{\bfseries\Huge Christopher Hughes}
\author{}
\date{}

\definecolor{lightgray}{gray}{0.8}
\newcolumntype{L}{>{\raggedleft}p{0.14\textwidth}}
\newcolumntype{R}{p{0.8\textwidth}}
\newcommand\VRule{\color{lightgray}\vrule width 0.5pt}


\begin{document}

	\newpage
	\setcounter{page}{1}
	\noindent

\begin{LARGE}

	\noindent\textbf{Christopher Hughes}\vspace{1pt}

\end{LARGE}


\noindent\rule{\textwidth}{1pt}\vspace{9pt}

\noindent
\begin{minipage}{0.5\textwidth}
	\begin{flushleft}
		chughes@bcgsc.ca\\
		Nationality: Canadian\\ \href{https://scholar.google.com/citations?user=jPSwBGwAAAAJ}{Google Scholar Citations}
	\end{flushleft}
\end{minipage}%
\begin{minipage}{0.5\textwidth}
	\begin{flushright}
		2477 Carolina Street, Unit 303\\
		V5T 0G8 Vancouver, Canada\\ \href{https://www.linkedin.com/in/christopher-hughes-612460133/}{LinkedIn Profile}
	\end{flushright}
\end{minipage}\vspace{6pt}


\section*{Relevant Skills}
\begin{addmargin}[3em]{2em}% 1em left, 2em right
	\subsection*{Research and Leadership Abilities}
	\begin{itemize}
		\item Highly skilled with a broad range of molecular biology techniques, including the manipulation and testing of bacteria and yeast using a variety of cloning and PCR techniques.
        \item Extensive experience using software tools such as Microsoft and Google Office suites, R, Python, and Adobe Illustrator for data logging, analysis, interpretation, reporting, and visalization.
        \item Experienced in the optimization, operation, and maintenance of a wide variety of highly-technical scientific instruments, such as mass spectrometers, HPLC hardware, PCR thermocyclers, and microplate readers. 
		\item Skilled in the development, management, and execution of small and large scale projects comprising individuals or collaborative teams of individuals working towards completion of set aims.
		\item I am motivated, a fast learner, excited to learn and contribute to part of a team, and am reliable.

		%\item Extensive experience working with a range mass spectrometry
		%architectures and associated hardwares (e.g. LC, UV) and softwares from
		%manufacturers such as Thermo, AB Sciex, Waters, and Agilent.

		%\item Strong leadership skills developed through management of numerous
		%undergraduate, graduate student, and technician research projects.
		%  \item Experienced working with proteomics in clinical settings using patient
		%  derived material, and with the design and execution of large-cohort studies.
		%  \item Significant exposure and experience with proteomics experiment design,
		%  execution, and data analysis, including shotgun, targeted (SRM),
		%  phosphorylation, and intact protein analyses.
		%  \item Extensive experience handling a broad range of mass spectrometer
		%  architectures and platforms, as well as accompanying chromatography systems.
		%  \item Highly skilled in the development of technologies and bioinformatic
		%  solutions for challenging proteomics applications, such as ultrasensitive
		%  analysis of quantity-limited samples and tissues.
		%  \item Experienced in the design, preparation, and bioinformatic analysis of
		%  samples for next-generation sequencing analysis, and their incoporation with
		%  proteomics.
	\end{itemize}
\end{addmargin}

\section*{Current Role}
\begin{tabular}{L!{\VRule}R}
	2018-present&\textbf{Staff Scientist}\\
	&British Columbia Cancer Research Centre\\
\end{tabular}\\

\begin{addmargin}[7.5em]{2em}% 1em left, 2em right
	In this role with Dr. Poul Sorensen, my responsibilities include:
	\begin{itemize}
		\item Independent conception, development, grant acquisition, execution, and supervision of studies examining the dynamics of protein translation at a sub-cellular level in cancer, with specific sub-studies focused on elucidating the mechanistic roles of targets using combinations of molecular biology, sequencing, proteomics, metabolomics, microscopy, and bioinformatic techniques.
		\item Participation in pan-Canada and global research projects involving the Sorensen lab, including experimental conception and design, data acquisition and analysis, grant acquisition, and student supervision.
	\end{itemize}
\end{addmargin}

\section*{Recent Publications}
{\setlength{\extrarowheight}{4pt}%
\begin{tabular}{L!{\VRule}R}
	2021&Zhang, H., \textbf{Hughes, C.S.} \textit{et al}, Proteomic screens for suppressors of anoikis identify IL1RAP as a promising surface target in Ewing sarcoma \textit{Cancer Discovery}. PMID: 34021002\\
    2019&\textbf{Hughes, C.S.}, Sorensen, P.H., Morin, G.B. A Standardized and Reproducible Proteomics Protocol for Bottom-up Quantitative Analysis of Protein Samples using SP3 and Mass Spectrometry \textit{Methods in Mol. Biol.}. PMID: 30852816\\
	2019&\textbf{Hughes, C.S.}, Moggridge, S., Mueller, T., Sorensen, P.H., Morin, G.B., Krijgsveld, J. Single-pot, Solid-phase-enhanced Sample Preparation for Proteomics Experiments \textit{Nature Protocols}. PMID: 30464214\\
	2019&Kovalchik, K.A., Colborne, S., Spencer, S., Sorensen, P.H., Chen, D.D.Y., Morin, G.B., \textbf{Hughes, C.S.\textsuperscript{$\diamond$}}, RawTools: Rapid and Dynamic Interrogration of Orbitrap Data Files for Mass Spectrometer System Management \textit{J. Prot. Res.}. PMID: 30462513\\
	2018&Kovalchik, K.A., Moggridge, S., Chen, D.D.Y., Morin, G.B., \textbf{Hughes, C.S.\textsuperscript{$\diamond$}} Parsing and Quantification of Raw Orbitrap Mass Spectrometer Data using RawQuant \textit{J. Prot. Res.}. PMID: 29682972\\
	2018&Moggridge, S., Sorensen, P.H., Morin, G.B., \textbf{Hughes, C.S.\textsuperscript{$\diamond$}} Extending the Compatibility of the SP3 Paramagnetic Processing Approach for Proteomics \textit{J. Prot. Res.}. PMID: 29565595\\
	2018&\textbf{Hughes, C.S.}, Morin, G. Using Public Data for Comparative Proteome Analysis in Precision Medicine Studies \textit{Proteomics}. PMID: 28887829\\
	2017&\textbf{Hughes, C.S.}, Spicer, V., Krokhin, O.V., Morin, G.B., Investigating Acquisition Performance on the Orbitrap Fusion When Using Tandem MS/MS/MS Scanning with Isobaric Tags \textit{J. Prot. Res.}. PMID: 28418257\\
	2017&\textbf{Hughes, C.S.}, Zhu, C., Spicer, V., Krokhin, O.V., Morin, G. Evaluating the Characteristics of Reporter Ion Signal Acquired
	in the Orbitrap Analyzer for Isobaric Mass Tag Proteome Quantification Experiments \textit{J. Prot. Res.}. PMID: 28418254\\
	2017&Tien, J.F., Mazloomian, A., Cheng, S.G., \textbf{Hughes, C.S.} \textit{et al}. CDK12 regulates alternative last exon mRNA splicing and promotes breast cancer cell invasion \textit{Nucleic Acids Res.}. PMID:28334900\\
	2017&Funnell, T., Tasaki, S., Oloumi, A., Araki, S., Kong, E., Yap, D., Nakayama, Y., \textbf{Hughes, C.S.} \textit{et al}. CLK-dependent exon
	recognition and conjoined gene formation revealed with a novel small molecule inhibitor \textit{Nat. Commun.}.PMID: 28232751\\
	2016&\textbf{Hughes, C.S.}, McConechy, M., Cochrane, D., Nazeran, T., Karnezis, A., Huntsman, D., Morin, G. Biomarker Discovery from High Resolution Proteomic Analysis of Fixed Ovarian Tumor Tissue Samples. \textit{Scientific Reports}. PMID: 27713570\\
	2016&Virant-Klun, I., Leicht, S., \textbf{Hughes, C.S.}, Krijgsveld, J. Identification of maturation-specific proteins by single-cell proteomics of human oocytes. \textit{Mol. Cell. Proteomics} PMID: 27215607\\

\end{tabular}

\newpage
\noindent
\section*{Patents}
\begin{tabular}{L!{\VRule}R}
	2014& \textbf{Title:} Proteomic sample preparation using paramagnetic
	beads\\
	&\textbf{Inventors:} \textbf{Hughes, C.S.}*, Krijgsveld, J.,
	Steinmetz, L.\\
	&\textbf{Publication number:} WO2015118152A1, US20170074869A1, CA2938907A1, EP3102612A1\\
	&\textbf{Filing date:} 2015-02-09\\
	&* - denotes majority inventor
\end{tabular}

\section*{Professional Experience}
\begin{tabular}{L!{\VRule}R}
    2014--2018&{\bf Group Leader and Proteomics Platform Manager}\\
	&{British Columbia Genome Sciences Centre}\\
	&Development, execution, analysis, and management of research projects utilizing proteomics in a diverse array of clinical experimental models, from large-scale patient cohort profiling to precision medicine clinical cancer trials.\\
    2012--2014&{\bf EIPOD Post-Doctoral Researcher}\\
	&{\bf European Molecular Biology Laboratory}\\
	&In-depth insight into the multi-layered regulation of cellular phenotype making use of protein turnover data acquired across a population of \textgreater200 individual yeast segregants.\\
	2006--2006&{\bf Research Technician}\\
	&{Merck Frosst Canada}\\
	&Worked on validating a new mass
	spectrometer ion source based on thermal desorption.\\
	2006--2006&{\bf Research Technician}\\
	&{Radient Technologies Inc.}\\
	&Worked on projects related to utilizing large-scale
	strategies to purify active compounds from natural products.\\
	2004--2005&{\bf Research Technician}\\
	&{MDS Sciex}\\
	&Worked on projects related to profiling carcinogenic food dyes and excipient analysis of analgesic drugs.\\
    2004--2004&{\bf Research Technician}\\
	&{Agriculture and Agri-Food Canada}\\
	&Worked on projects related to profiling resistance of soybean varities to different pathogens.\\
\end{tabular}

\section*{Education}
\begin{tabular}{L!{\VRule}R}
	2007--2012&{\bf PhD in Biochemistry, Western University, Canada}\\
	&Thesis title: Proteomics of Human Stem Cell Derived Matrices\\
	2002--2007&Honors Bachelor of Science in Biology, University of Waterloo, Canada\\
	%2013--2013&Next-generation Sequencing Workshop, European Bioinformatics Institute, UK\\
	%2013--2013&Introduction to Python, EMBL, Germany\\
	%2013--2013&Introduction to MATLAB and image analysis, EMBL, Germany\\
	%2010--2010&Stem Cell Culture Training Course, Stem Cell Technologies, Canada\\
	%2009--2009&Quantitative Proteomics Workshop, American Society for Mass
	%Spectrometry\\
	%2008--2008&FT-ICR Mass Spectrometry Workshop, American Society for Mass
	%Spectrometry\\
\end{tabular}

\section*{References}
Additional references available upon request.\\

\end{document}
