\documentclass[11pt]{article}
\usepackage{array, xcolor, lipsum, bibentry, parallel}
\usepackage[margin=3cm]{geometry}
\usepackage{setspace}
\usepackage{tabularx}
\usepackage{paralist}
\usepackage{scrextend}
\usepackage[colorlinks = true, linkcolor = blue, urlcolor  = blue, citecolor = blue, anchorcolor = blue]{hyperref}


\title{\bfseries\Huge Christopher Hughes}
\author{}
\date{}

\definecolor{lightgray}{gray}{0.8}
\newcolumntype{L}{>{\raggedleft}p{0.14\textwidth}}
\newcolumntype{R}{p{0.8\textwidth}}
\newcommand\VRule{\color{lightgray}\vrule width 0.5pt}


\begin{document}

	\newpage
	\setcounter{page}{2}
	\noindent

\begin{LARGE}

	\noindent\textbf{Christopher Hughes}\vspace{1pt}

\end{LARGE}


\noindent\rule{\textwidth}{1pt}\vspace{9pt}

\noindent
\begin{minipage}{0.5\textwidth}
	\begin{flushleft}
		christopher.hughes@dal.ca\\
		Nationality: Canadian\\ 
		\href{https://scholar.google.com/citations?user=jPSwBGwAAAAJ}{Google Scholar Citations}\\
		\href{https://github.com/chrishuges}{GitHub}
	\end{flushleft}
\end{minipage}%
\begin{minipage}{0.5\textwidth}
	\begin{flushright}
		6144 Willow Street\\
		B3K 1M2 Halifax, Canada\\ 
		\href{https://www.linkedin.com/in/christopher-hughes-612460133/}{LinkedIn Profile}
	\end{flushright}
\end{minipage}\vspace{6pt}


\section*{Relevant Skills}
\begin{addmargin}[3em]{2em}% 1em left, 2em right
	\subsection*{Research and Leadership Abilities}
	\begin{itemize}
		%\item Expertise in the operation, maintenance, and repair of HPLC systems from a variety of vendors, including troubleshooting a variety of issues stemming from the HPLC itself, or the hardware it operates in cooperation with.
        %\item Extensive experience with all aspects of the development and continuous operation of a mass spectrometry core facility, including: instrument acquisition, maintenance, repair, and optimization; protocol optimization and SOP development; day-to-day project management for a large and diverse customer base; construction of cost models and financial management; preparation of applications to pursue funding initiatives.
        \item Experience working with a broad range of clinical samples, including tissues and fluids, to facilitate their examination for drug tracking, biomarker discovery, or broader biological analyses. 
        \item Skilled in the application of proteomics, metabolomics, DNA/RNA sequencing, and other technologies (e.g. microscopy, flow cytometry, mass cytometry, other molecular biology methods such as qPCR, Western blotting) towards the study of complex biological systems.
        \item Extensive expertise with the operation, maintenance, and utilization of mass spectrometers towards their application in the study of a broad range of molecular systems. 
        \item Expertise in the use bioinformatic languages, such as R, Python, and C\#, for the analysis of proteomics, DNA/RNA sequencing, and metabolomics data. Proficient with Adobe Illustrator for the creation of high quality figures and rich schematics.
        \item Experience transitioning project ideas and preliminary findings into complete applications for granting agencies, such as CIHR, CFI, NSERC, CRS, Alex's Lemonade Stand, and NIH.		
		\item Skilled in the conception, development, management, and execution of small and large scale research projects comprising individuals or collaborative teams of researchers working towards completion of set aims.
		%\item Direct exposure to all aspects of the management and operation of a core facility that caters to a wide range of individuals, from researchers to clinicians.
		%\item Strong leadership and training skills developed through management of numerous undergraduate, graduate student, and technician research projects.
	\end{itemize}
\end{addmargin}

\section*{Current Role}
\begin{tabular}{L!{\VRule}R}
	2023-present&\textbf{Manager, Biological Mass Spectrometry Core}\\
	&Dalhousie University\\
\end{tabular}\\

\begin{addmargin}[7.5em]{2em}% 1em left, 2em right
	In this role, I am tasked with all aspects of management of a mass spectrometry core facility catering to a wide range of individuals researching a diverse set of topics. My responsibilities include study conception and design, instrument operation/maintenance/calibration/repair, funding acquisition, project management, facility management, training and knowledge dissemination, and personnel management. 
    %In this role with Dr. Poul Sorensen, my primary responsibilities include the independent conception, development, funding acquisition for, execution, and supervision of studies examining mechanisms of cancer cell adaptation mediated by modification of mRNA translation, with a particular emphasis on the RNA-binding protein YB-1. Studying the oncogenic properties of Ewing sarcoma with a specific focus on patterns in the expression of transcript and protein isoforms specific to this cancer type, with additional effort focused on the putative tumour suppressor DLG2.
		%\item Participation in numerous pan-Canada and global research projects involving the Sorensen lab (e.g. PROFYLE, SU2C), including experimental conception and design, data acquisition and analysis, grant acquisition, and student supervision.
\end{addmargin}


\section*{Selected Publications}
{\setlength{\extrarowheight}{4pt}%
\begin{tabular}{L!{\VRule}R}
    2024&Lizardo, M., \textbf{Hughes, C.S.} \textit{et al}, Pharmacologic inhibition of eIF4A blocks NRF2 synthesis to prevent osteosarcoma metastasis \textit{Clin. Cancer Research}. PMID: 39078310\\
    %2024&Mooney, B., Negri, G.L., Shyp, T., Delaidelli, A., Zhang, H., Spencer, S.E., Weiner, A.K., Radoui, A.B., Shraim, R., Lizardo, M.M., \textbf{Hughes, C.S.} \textit{et al}, Surface and Global Proteome Analyses Identify ENPP1 and Other Surface Proteins as Actionable Immunotherapeutic Targets in Ewing Sarcoma \textit{Clin. Cancer Research}. PMID: 37812652\\
    2023&Zhang, H.F., Delaidelli, A., Javed, S., Turgu, B., Morrison, T., \textbf{Hughes, C.S.} \textit{et al} A MYCN-independent mechanism mediating secretome reprogramming and metastasis in MYCN-amplified neuroblastoma \textit{Science Advances}. PMID: 37611092\\
    2023&Johnson, F.D., \textbf{Hughes, C.S.} \textit{et al}, Tandem mass tag-based thermal proteome profiling for the discovery of drug-protein interactions in cancer cells \textit{STAR Protocols}. PMID: 36856765\\
    2023&Asleh, K., Negri, G.L., Spencer, S.E., Colborne, S., \textbf{Hughes, C.S.} \textit{et al} Proteomic analysis of archival breast cancer clinical specimens identifies biological subtypes with distinct survival outcomes \textit{Nat. Communications}. PMID: 35173148\\
	2021&Zhang, H., \textbf{Hughes, C.S.} \textit{et al}, Proteomic screens for suppressors of anoikis identify IL1RAP as a promising surface target in Ewing sarcoma \textit{Cancer Discovery}. PMID: 34021002\\
	2019&\textbf{Hughes, C.S.}, Sorensen, P.H., Morin, G.B. A Standardized and Reproducible Proteomics Protocol for Bottom-up Quantitative Analysis of Protein Samples using SP3 and Mass Spectrometry \textit{Methods in Mol. Biol.}. PMID: 30852816\\
	2019&\textbf{Hughes, C.S.}, Moggridge, S., Mueller, T., Sorensen, P.H., Morin, G.B., Krijgsveld, J. Single-pot, Solid-phase-enhanced Sample Preparation for Proteomics Experiments \textit{Nature Protocols}. PMID: 30464214\\
	2019&Kovalchik, K.A., Colborne, S., Spencer, S., Sorensen, P.H., Chen, D.D.Y., Morin, G.B., \textbf{Hughes, C.S.\textsuperscript{$\diamond$}}, RawTools: Rapid and Dynamic Interrogration of Orbitrap Data Files for Mass Spectrometer System Management \textit{J. Prot. Res.}. PMID: 30462513\\
	2018&Moggridge, S., Sorensen, P.H., Morin, G.B., \textbf{Hughes, C.S.\textsuperscript{$\diamond$}} Extending the Compatibility of the SP3 Paramagnetic Processing Approach for Proteomics \textit{J. Prot. Res.}. PMID: 29565595\\
	2018&\textbf{Hughes, C.S.}, Morin, G. Using Public Data for Comparative Proteome Analysis in Precision Medicine Studies \textit{Proteomics}. PMID: 28887829\\
	2017&\textbf{Hughes, C.S.}, Spicer, V., Krokhin, O.V., Morin, G.B., Investigating Acquisition Performance on the Orbitrap Fusion When Using Tandem MS/MS/MS Scanning with Isobaric Tags \textit{J. Prot. Res.}. PMID: 28418257\\
	2017&\textbf{Hughes, C.S.}, Zhu, C., Spicer, V., Krokhin, O.V., Morin, G. Evaluating the Characteristics of Reporter Ion Signal Acquired
	in the Orbitrap Analyzer for Isobaric Mass Tag Proteome Quantification Experiments \textit{J. Prot. Res.}. PMID: 28418254\\
	%2016&\textbf{Hughes, C.S.}, McConechy, M., Cochrane, D., Nazeran, T., Karnezis, A., Huntsman, D., Morin, G. Biomarker Discovery from High Resolution Proteomic Analysis of Fixed Ovarian Tumor Tissue Samples. \textit{Scientific Reports}. PMID: 27713570\\
	%2014&\textbf{Hughes, C.S.}, Foehr, S., Garfield, D., Furlong, E.E., Steinmetz, L.M., Krijgsveld, J. Ultrasensitive proteome analysis using paramagnetic bead technology. \textit{Molecular Systems Biology}. PMID: 25358341\\
	%2012&\textbf{Hughes,C.S.} and Krijgsveld, J. Developments in quantitative mass spectrometry for the analysis of proteome dynamics. \textit{Trends in Biotechnology}. PMID: 23107010\\
	%2012&\textbf{Hughes, C.S.} \textit{et al}. Mass spectrometry-based proteomic analysis of the matrix microenvironment in pluripotent stem cell culture.\textit{Mol. Cell. Prot.}. PMID: 23023296\\
	&$\diamond$ - denotes senior authorship.\\
\end{tabular}

\bigskip

\section*{Selected Grants and Awards}
{\setlength{\extrarowheight}{4pt}%
\begin{tabular}{L!{\VRule}R}
	2024& Implementing a pan-Canadian standardized mass spectrometry-based proteomics approach for the analysis of clinical cancer samples: validation using archived breast cancer specimens for biomarker discovery. \textbf{Agency:} Terry Fox Marathon of Hope, \textbf{Role:} co-applicant\\
    2024& Investigating mechanistic roles of a novel onco-fusion driven transcript isoform in Ewing sarcoma. \textbf{Agency:} Alex's Lemonade Stand Innovation Fund, \textbf{Role:} co-applicant\\
    2022& Examining onco-fusion-driven expression of transcript and protein isoforms that underpin fitness relationships essential for Ewing sarcoma tumor formation. \textbf{Agency:} Sarcoma Foundation of America, \textbf{Role:} co-applicant\\
	2021& Ewing sarcoma progression and spreading are controlled by selected protein production events that are regulated by modified forms of the YB-1 oncoprotein. \textbf{Agency:} MGI, \textbf{Role:} co-applicant\\
	2016& Fixed tissue proteomics (FTP) applied to create a pragmatic clinical decision aid for endometrial cancer. \textbf{Agency:} CCSRI, \textbf{Role:} co-applicant\\
	2016& Breast cancer classification and marker identification by comprehensive proteomic analysis. \textbf{Agency:} CBCF, \textbf{Role:} co-applicant\\
	2016& High-resolution analysis of phenotypic fitness using genome-wide CRISPR editing coupled to quantitative mass spectrometry. \textbf{Agency:} BCPN, \textbf{Role:} co-applicant\\
	%2014&European Interdisciplinary Post-Doctoral Fellowship\\
	%2009&NSERC Canadian Graduate Scholarship - Doctoral\\
	%2008&NSERC Canadian Graduate Scholarship - Masters\\
\end{tabular}

\section*{Selected Presentations}
{\setlength{\extrarowheight}{4pt}%
\begin{tabular}{L!{\VRule}R}
	Poster&CCRC 2019 \textbf{Title:} A subcellular atlas of translation machinery reveals novel roles for the RNA binding protein YB-1\\
	Oral&Personalized Oncogenomics Clinical Research Session 2016 \textbf{Title:} Proteomics and Metabolomics in Personalized Oncogenomics\\
	Oral&BCPN 2016 \textbf{Title:} High Resolution Proteomic Analysis of Ovarian FFPE Tumour Tissues using TMT-MS3 on an Orbitrap Fusion for Clinical Research\\
	Poster&ASMS 2015 \textbf{Title:} Enhanced processing of FFPE tissue for clinical proteomics using SP3\\
	Poster&ASMS 2014 \textbf{Title:} Single-tube sample preparation workflows for Ultra-sensitive Proteomics\\
	Oral&Nordic Proteomics Meeting 2014 \textbf{Title:} Single-tube sample preparation workflows for Ultra-sensitive Proteomics\\
	Poster&Proteostasis Discussion 2013 \textbf{Title:} Studying the dynamics of proteome homeostasis using Hyperplexed mass spectrometry\\
	%Poster&HUPO 2011 \textbf{Title:} Analysis of the Human Embryonic Stem Cell Depositome\\
	%Poster&ISSCR 2011 \textbf{Title:} Proteomic Analysis of the Human Embryonic Stem Cell Depositome\\
	%Poster&HUPO 2010 \textbf{Title:} Matrigel: A complex mixture required for optimal growth of cell culture\\
	%Poster&HUPO 2010 \textbf{Title:} Analysis and quantitation of a glycogen synthase kinase-3 knockout embryonic stem cell line to a depth of 3500 proteins\\
	%Poster&ASMS 2009 \textbf{Title:} Analysis and quantitation of a glycogen synthase kinase-3 knockout embryonic stem cell line to a depth of 3500 proteins\\
	%Oral&ASMS 2008 \textbf{Title:} Prevention of amino acid conversion in SILAC experiments with embryonic stem cells\\
\end{tabular}

\section*{Patents}
{\setlength{\extrarowheight}{4pt}%
\begin{tabular}{L!{\VRule}R}
	2014& \textbf{Title:} Proteomic sample preparation using paramagnetic
	beads\\
	&\textbf{Inventors:} \textbf{Hughes, C.S.}*, Krijgsveld, J.,
	Steinmetz, L.\\
	&\textbf{Publication number:} WO2015118152A1, US20170074869A1, CA2938907A1, EP3102612A1\\
	&\textbf{Filing date:} 2015-02-09\\
	&* - denotes majority inventor
\end{tabular}

\section*{Selected Education and Professional Experience}
{\setlength{\extrarowheight}{4pt}%
\begin{tabular}{L!{\VRule}R}
	2018--2023&{Staff Scientist and Mass Spectrometry Service Specialist, }{\bf Sorensen lab, BC Cancer}\\
	&Primary responsibilities included the independent conception, development, funding acquisition for, execution, and supervision of studies examining mechanisms of cancer cell adaptation mediated by modification of mRNA translation.\\
    2014--2018&{Group Leader, }{\bf Genome Sciences Centre, BC Cancer}\\
	&Managed the Mass Spectrometry Proteomics Platform. Involved the development, execution, analysis, and management of research projects utilizing mass spectrometry to study a range of experimental models. Developed and optimized protocols and SOPs for a wide variety of sample analysis types that cater to the facility user base. Optimized, maintained, and repaired hardware housed in the core facility. Managed administrative and financial aspects of the core facility. Prepared grant applications for funding to acquire new equipment and support research and other technical development projects.\\
	2012--2014&{Post-Doctoral Researcher, }{\bf European Molecular Biology Laboratory}\\
	&Performed integrative studies that utilized genomics, transcriptomics, and proteomics to study dynamic molecular systems.\\
	2007--2012&{PhD in Biochemistry, }{\bf Western University, Canada}\\
	%&Thesis title: Proteomics of Human Stem Cell Derived Matrices\\
	%2006--2006&{Research Technician, }{\bf Merck Frosst Canada}\\
	%&Worked with the drug metabolism team to test and validate of a new mass
	%spectrometer ion source based on thermal desorption.\\
	%2006--2006&{Research Technician, }{\bf Radient Technologies Inc.}\\
	%&Worked in the drug purification team utilizing large-scale
	%strategies to purify active compounds from natural products.\\
	%2004--2005&{Research Technician, }{\bf MDS Sciex}\\
	%&Worked within the Demo and Projects Laboratories on projects related to
	%profiling carcinogenic food dyes and excipient analysis of analgesic drugs.\\
	%2002--2007&{Honors Bachelor of Science in Biology, }{\bf University of Waterloo, Canada}\\
\end{tabular}


%\section*{References}
%Additional references available upon request.\\
%\begin{addmargin}[3em]{2em}% 1em left, 2em right

%	\noindent\textbf{Dr. Poul Sorensen}\\
%	\textit{British Columbia Cancer Agency, Vancouver, Canada}\\
%	Role: Senior Scientist\\
%	Email: psorensen@bccrc.ca\\
%	Phone: +1 604-675-8202\\
%	Relationship: Current supervisor\\

%	\noindent\textbf{Dr. Gregg Morin}\\
%	\textit{British Columbia Cancer Agency, Vancouver, Canada}\\
%	Role: Head of Proteomics\\
%	Email: gmorin@bcgsc.ca\\
%	Phone: +1 604-675-8154\\
%	Relationship: Previous supervisor\\

	%\noindent\textbf{Dr. Jeroen Krijgsveld}\\
	%\textit{German Cancer Research Centre, Heidelberg, Germany}\\
	%Role: Group leader, Proteomics of Stem Cells and Cancer\\
	%Email: j.krijgsveld@dkfz-heidelberg.de\\
	%Phone: +49 06221 421720\\
	%Relationship: Post-doctoral supervisor\\

	%\noindent\textbf{Dr. Lars Steinmetz}\\
	%\textit{European Molecular Biology Laboratory, Heidelberg, Germany}\\
	%Role: Group leader\\
	%Email: lars.steinmetz@embl.de\\
	%Phone: +49 06221 387-8389\\
	%Preferred contact method: Email\\
	%Relationship: Post-Doctoral Supervisor\\

%	\noindent\textbf{Dr. Gilles Lajoie}\\
%	\textit{Western University, London, Canada}\\
%	Role: Group leader and head of Proteomics Core Facility\\
%	Email: galajoie@gmail.com\\
%	Phone: +1 519-661-3054 ext.83054\\
%	Relationship: Ph.D. supervisor
%\end{addmargin}
\end{document}
